locations/suedblock.yml

```diff
@@ -0,0 +1,62 @@
+---
+location: suedblock
+location_nice: Suedblock
+latitude: 52.498599118
+longitude: 13.416844010
+contact_nickname: '365ff'
+contacts:
+  - '365ff [ät] systemli [dot] org'
```

Comment:
Are you @pktpls the new maintainer of this location? If so please change the contact field